pkg/uki/install.go

```diff
@@ -120,6 +124,52 @@ func (i *InstallAction) Run() (err error) {
 		return err
 	}
 
+	// Remove entries
+	// Read all confs
+	err = fsutils.WalkDirFs(i.cfg.Fs, filepath.Join(i.spec.Partitions.EFI.MountPoint), func(path string, info os.DirEntry, err error) error {
+		if info.IsDir() {
+			return nil
+		}
+		if filepath.Ext(info.Name()) != ".conf" {
+			return nil
+		}
+		i.cfg.Logger.Debugf("Checking conf file %s", path)
+		// Extract the values
+		conf, err := godotenv.Read(path)
+		if err != nil {
+			return err
+		}
+		i.cfg.Logger.Debugf("Conf file %s has values %v", path, litter.Sdump(conf))
+		if len(conf["cmdline"]) == 0 {
+			return nil
+		}
+		// Check if the cmdline matches any of the entries in the skip list
+		for _, entry := range i.spec.SkipEntries {
+			// Match the cmdline key against the entry
+			if strings.Contains(conf["cmdline"], entry) {
+				i.cfg.Logger.Debugf("Found match for %s in %s", entry, path)
+				// If match, get the efi file and remove it
+				if conf["efi"] != "" {
+					i.cfg.Logger.Debugf("Removing efi file %s", conf["efi"])
+					// First remove the efi file
+					err = i.cfg.Fs.Remove(filepath.Join(i.spec.Partitions.EFI.MountPoint, conf["efi"]))
+					if err != nil {
+						return err
+					}
+					// Then remove the conf file
+					i.cfg.Logger.Debugf("Removing conf file %s", path)
+					err = i.cfg.Fs.Remove(path)
+					if err != nil {
+						return err
+					}
+					// Do not continue checking the conf file, we already done all we needed
+					break
```

Comment:
Doesn't this mean, we always remove one entry only? This `break` breaks the for loop in case a match is found. What if there was another match?

Reply:
This breaks the inner loop of checking the conf against the entries, not the walk dir function, no?

Reply:
yes, this is the inner loop, so once we match and remove the entry we skip going over the list of skip entries and continue the outer loop of the walkfs

Reply:
oh true, my bad

Reply:
umm maybe this one does exit the full loop somehow lmao